learningresources/tests/test_models.py

```diff
@@ -3,18 +3,79 @@
 """
 from __future__ import unicode_literals
 
+import os
+import random
+import string
+from shutil import rmtree
+from tempfile import mkdtemp
+
+from django.core.files import File
 from mock import MagicMock
 
 from .base import LoreTestCase
 from learningresources.models import (
+    FILE_PATH_MAX_LENGTH,
     LearningResourceType,
     Repository,
-    static_asset_basepath
+    static_asset_basepath,
+    StaticAsset,
+    FilePathLengthException
 )
 
 
+def get_random_string(length):
+    """Helper function to generate strings with random characters"""
+    return ''.join(
+        random.SystemRandom().choice(
```

Comment:
Is this random number generator deterministic? If we're using it in tests it should be so that we can repeat failures reliably. Please use `SystemRandom(0)` for that purpose